feature-polkaswap-impl/src/main/kotlin/jp/co/soramitsu/polkaswap/impl/presentation/swap_tokens/SwapTokensFragment.kt

```diff
@@ -52,13 +52,6 @@ class SwapTokensFragment : BaseComposeBottomSheetDialogFragment<SwapTokensViewMo
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        var constantDiff = 0
-        view.postDelayed({
-            val w = Rect()
-            view.getWindowVisibleDisplayFrame(w)
-            constantDiff = view.rootView.height - (w.bottom - w.top)
-        }, 100)
-
         view.viewTreeObserver.addOnGlobalLayoutListener {
```

Comment:
maybe completely remove this?

Reply:
done as complete as possible; leave soft keyboard visibility checks only